main.h

```diff
@@ -36,21 +35,18 @@ class Items{
 };
 
 std::ostream& operator<<(std::ostream& os, const Items& item){
-  os << item.call_code << "- " << item.name << std::endl;
-  return os;
+  return os << item.call_code << "- " << item.name;
 }
 
 class Menu {
   public:
   void AddItem(const std::string name, const int call_code) {
-    Items new_item_temporal(name, call_code);
-    items.emplace_back(new_item_temporal);
+    items.emplace_back(name, call_code);
   }
```

Comment:
nit: Insert linebreak between functions.
```suggestion
  }

```